src/vips-notification-worker/src/main/java/ca/bc/gov/open/pssg/rsbc/vips/notification/worker/sftp/SftpServiceImpl.java

```diff
@@ -0,0 +1,76 @@
+package ca.bc.gov.open.pssg.rsbc.vips.notification.worker.sftp;
+
+import com.jcraft.jsch.ChannelSftp;
+import com.jcraft.jsch.JSchException;
+import com.jcraft.jsch.Session;
+import com.jcraft.jsch.SftpException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class SftpServiceImpl implements SftpService {
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    private final Session session;
+
+    public SftpServiceImpl(Session session) {
+        this.session = session;
+    }
+
+    public ByteArrayInputStream getContent(String remoteFilename) throws DpsSftpException {
+
+        ByteArrayInputStream result = null;
+        ChannelSftp channelSftp = null;
+
+        byte[] buff = new byte[8000];
```

Comment:
do we need to parameter-ize the 8000 here?

Reply:
yeah usually you go a bit under 1024 * 8, I can set it as a constant right....